### PR TITLE
Sliding sync hang fix

### PR DESCRIPTION
**Pull request by stefanceriu**
This PR fixes sliding sync not restarting properly when coming back from an app suspended mode. When that happens scrolling the list results in more range requests that the internal channel can support and blocks the main thread indefinitely

### CODE REVIEW

File: ElementX/Sources/Application/AppCoordinator.swift [hunk 1]
```diff
@@ -436,19 +436,22 @@ class AppCoordinator: AppCoordinatorProtocol, AuthenticationCoordinatorDelegate,
...
-            
-            if reachable {
-                ServiceLocator.shared.userIndicatorController.retractIndicatorWithId(reachabilityNotificationIdentifier)
-            } else {
-                ServiceLocator.shared.userIndicatorController.submitIndicator(.init(id: reachabilityNotificationIdentifier,
-                                                                                    title: L10n.commonOffline,
-                                                                                    persistent: true))
+        networkMonitorObserver = ServiceLocator.shared.networkMonitor
+            .reachabilityPublisher
+            .removeDuplicates()
```
Comment (pixlwave) [inline ElementX/Sources/Application/AppCoordinator.swift:441]: Good spot

**Review by pixlwave — APPROVED** [review]
LGTM

File: ElementX/Sources/Application/AppCoordinator.swift (hunk 1 above)
Comment (stefanceriu) [reply ElementX/Sources/Application/AppCoordinator.swift:441]: Didn't really affect anything, just a bunch of useless logs

**Comment by ara4n** [comment]
would this explain https://github.com/vector-im/element-x-ios-rageshakes/issues/389?

**Comment by ara4n** [comment]
answer: nope, as in that rageshake the app UI hasn't locked up.

**Pull request merged**